components/HistoryTable.qml

```diff
@@ -52,6 +54,22 @@ ListView {
             + translationManager.emptyString;
     }
 
+    function lookupPaymentID(paymentId) {
```

Comment:
index and rowID are the same. First implementation used std::map and "rowId" was stored in object. After switching to std::vector storing of rowId isn't necessary and ought to be removed.